### PR TITLE
rename USB parser solutions

### PR DESCRIPTION
**Pull request by japaric**
also put them outside the `src` directory so people don't think they are modules

### CODE REVIEW

File: embedded-workshop-book/src/setup-stage.md [hunk 1]
```diff
@@ -25,16 +25,16 @@ You will also find this information in the `// TODO implement ...` comment in th
 
 To complete the task, proceed like this:
 
-1. **Parse GET_DESCRIPTOR requests:**  
+1. **Parse GET_DESCRIPTOR requests:**
```
Comment (Lotterleben) [inline embedded-workshop-book/src/setup-stage.md:28]: I'd like to keep the line break here if possible

File: embedded-workshop-book/src/setup-stage.md [hunk 2]
```diff
@@ -25,16 +25,16 @@ You will also find this information in the `// TODO implement ...` comment in th
...
 
     - check table 9-4 in the USB specification for Request Codes
     - remember that you can define binary literals by prefixing them with `0b`
     - you can use bit shifts (`>>`) and casts (`as u8`) to get the high/low bytes of a `u16`
 
-See `advanced/common/usb/src/get-descriptor-device.rs` for a solution.
+See `advanced/common/usb/solution-get-descriptor-device.rs` for a solution.
 
-2. **Read incoming request information and pass it to the parser:**  
+2. **Read incoming request information and pass it to the parser:**
```
Comment (Lotterleben) [inline embedded-workshop-book/src/setup-stage.md:37]: same here

**Comment by japaric** [comment]
@Lotterleben changed to keep the trailing spaces in both list items.

**Comment by Lotterleben** [comment]
awesome, thx. ✅ from me

**Pull request merged**